packages/core/src/controllers/ApiController.ts

```diff
@@ -65,6 +65,14 @@ export const ApiController = {
     }
   },
 
+  _filterOutExtensions(wallets: WcWallet[]) {
+    if (OptionsController.state.isUniversalProvider) {
+      return wallets.filter(w => Boolean(w.mobile_link))
```

Comment:
Why we only enable mobile wallets if universal provider enabled?

Reply:
updated to Mobile + Desktop + Webapps